enf-reducer/tests/zero-constraint-subsumption-test.metta

```diff
@@ -0,0 +1,14 @@
+! (register-module! ../../../metta-moses-reduction) 
+! (import! &self metta-moses-reduction:enf-reducer:rte-helpers)
+! (import! &self metta-moses-reduction:enf-reducer:zero-constraint-subsumption)
+ 
+ ! (assertEqualToResult (zeroConstraintSubsume (OR (AND A B) (AND)) (OR (AND A B) (AND))) (()))
+ ! (assertEqualToResult (zeroConstraintSubsume (OR (AND A B) (OR)) (OR (AND A B) (OR))) (()))
+ ! (assertEqualToResult (zeroConstraintSubsume (OR (AND A B) (A)) (OR (AND A B) (A))) ((OR (AND A B) (A))))
```

Comment:
Why did you write the `(A)` in the `(OR (AND A B) (A))` test case? Do we have a situation where a literal being enclosed within a tuple?

Reply:
I don't think there is, I was trying all possible combinations. Should I remove it? also, is (OR) a possibility?